.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 1 * * *'
```

Comment:
Could you add the a comment with the equivalent period this means in "human language". I must admit that I suffer a lot with cronjob syntax.

Reply:
Comment already included